hmpps-prisoner-search-indexer/RegressionTests.md

```diff
@@ -2,33 +2,36 @@
 
 Recommended regression tests is as follows:
 
-* A partial build of index - see the `Rebuilding an index` instructions below. The rebuild does not need to be completed but expect the info to show something like this:
+* A partial build of index - see the `Rebuilding an index` instructions below. The rebuild does not need to be 
```

Comment:
only formatting here - will update this properly next as still has the old information in it